packages/apps/job-launcher/server/src/modules/job/job.service.spec.ts

```diff
@@ -601,11 +620,26 @@ describe('JobService', () => {
       getUserBalanceMock.mockResolvedValue(userBalance);
 
       await expect(
-        jobService.createJob(userId, JobRequestType.FORTUNE, fortuneJobDto),
+        jobService.createJob(userMock, JobRequestType.FORTUNE, fortuneJobDto),
       ).rejects.toThrow(
         new ControlledError(ErrorJob.NotEnoughFunds, HttpStatus.BAD_REQUEST),
       );
     });
+
```

Comment:
Missing:
- Should create a job if the user is whitelisted.
- Should create a job id the user is not whitelisted and has a credit card.

How is possible that other test cases work if:
- isUserWhitelisted is not mocked
- getDefaultPaymentMethod is not mocked